packages/api/src/domain/webhook.ts

```diff
@@ -0,0 +1,17 @@
+export const dapiWHPrefix = "devices";
```

Comment:
nit: any reason not to use `Product.devices` here?

Reply:
No, didn't think of that. Its a coincidence those ended up being the same, one was used for webhooks and the other for usage reporting. But I can update it. 👍